integration-tests/test_unregister.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+
+pytestmark = pytest.mark.usefixtures("register_subman")
+
+
+def test_unregister(insights_client):
+    """Test insights-client --unregister after registering it
+
+    If the unregistration is successful, `unregister` method returns True
+    """
+    insights_client.register()
+    assert insights_client.is_registered
+
+    registration_status = insights_client.run("--unregister")
```

Comment:
```suggestion
    registration_status = insights_client.unregister()
```

Reply:
In my opinion I do not see a problem with this, also it looks symmetrical throughout test(aesthetically )